### PR TITLE
Build multi-platform Docker images

### PR DESCRIPTION
**Pull request by ricekot**
Part of #6944.

### CODE REVIEW

File: .github/workflows/release-main-docker.yml [hunk 1]
```diff
@@ -15,39 +15,41 @@ jobs:
...
-        name: Build bare Docker image
-        uses: docker/build-push-action@v2
+        name: Build and push bare Docker image
+        uses: docker/build-push-action@v3
         with:
-          load: true
           context: docker
           file: docker/Dockerfile-bare
+          platforms: linux/amd64,linux/arm64
+          push: true
```
Comment (thc202) [inline .github/workflows/release-main-docker.yml:51]: Is the push done in the order of the declared tags?

Comment (thc202) [reply .github/workflows/release-main-docker.yml:51]: Answered in IRC.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by thc202** [comment]
This could be mentioned in the changelog.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**